Revert toggling of CONFIG_LINUX_PERF to fix aarch64 build
https://hg.mozilla.org/mozilla-central/rev/c174bd2624e2d4fc41721a5a51a48fbeae1666c2#l2.616

Index: media/ffvpx/config_unix_aarch64.h
--- media/ffvpx/config_unix_aarch64.h.orig
+++ media/ffvpx/config_unix_aarch64.h
@@ -615,7 +615,7 @@
 #define CONFIG_AUTODETECT 0
 #define CONFIG_FONTCONFIG 0
 #define CONFIG_LARGE_TESTS 1
-#define CONFIG_LINUX_PERF 1
+#define CONFIG_LINUX_PERF 0
 #define CONFIG_MACOS_KPERF 0
 #define CONFIG_MEMORY_POISONING 0
 #define CONFIG_NEON_CLOBBER_TEST 0
